docs/catalog.rst

```diff
@@ -1,5 +1,41 @@
 .. _sec_catalog:
 
+With ``stdpopsim``, you can run simulations from a number of bespoke demographic models
+that were chosen from notable manuscripts. These models have been rigorously checked 
+and tested by multiple people, so you can rest easy knowing that your simulations are
+reproducible and bug-free!
+
+This catalog shows you all of the possible bespoke simulations that
```

Comment:
Maybe drop "bespoke" here?

Reply:
agreed

Reply:
also rather than "chosen from notable manuscripts", how about a change to something like "implemented from published demographic histories"